db/migrate/20160329184508_create_gem_download_count_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateGemDownloadCountIndex < ActiveRecord::Migration
+  def change
+    add_index :gem_downloads, [:version_id, :rubygem_id, :count]
```

Comment:
> But since count will be changing a lot, the index will be changing a lot. And potentially render the table statistics invalid. So the index would not be very effective, and the data page hit may actually be quicker than dealing with a stale index.

That might change every 5 minutes. and only for gems that gets downloaded a lot. So I think the index is good in that case.

> The count column may slow down updates as well.

Yeah, that happens in a job, so it is fine.

> version_id is almost always be 0 - which is not good at the front of an index. Putting values that are the most different at the front makes for better data distribution

I want if this index, be able to look-up things by version_id only too. So thats why I added in the beginning. As far as i understand, there will be big branch on the b-tree index for the version_id = 0 , however the rest of the index will be balanced, as the second item is rubygem_id